spec/controllers/items_controller_spec.rb

```diff
@@ -102,8 +102,8 @@
     context 'with no name' do
       it 'populates a danger flash message' do
         post :create_item, params: { name: nil, type: item_type.name }
-        expect(flash[:warning]).to be_present
-        expect(flash[:warning]).to eq('Name is a required field')
+        expect(flash[:danger]).to be_present
```

Comment:
Not a blocking concern, I just want to mention that checking that `flash[:danger]` is eql to a string is definitely also checking that it's present.